docs/jwst/rscd/reference_files.rst

```diff
@@ -66,3 +41,55 @@ It uses ``EXTNAME=RSCD`` and contains seventeen columns:
 * SAT_ROWTERM: float
 * SAT_SCALE: float
 
+In order to explain where these parameters are used in the correction we will go over the correction equations given
+in the Description Section.
+
+The general form of the the correction to be added to the input data is::
+
+   corrected data = input data data + dn_accumulated * scale * exp(-T / tau)  (Equation 1)
+
+where T is the time since the last group in the last integration, tau is the exponential time constant found in the RSCD table
```

Comment:
Same comment as earlier about "time since the last group in the last integration".

Also, in general I'm not sure all of this needs to be repeated again in this section. All this section really needs to cover is the format and content of the ref file. You can just refer the reader to the Description section to find out how each parameter in the table is used.